harness/features/variable.cloud.test.ts

```diff
@@ -19,7 +19,7 @@ const scope = getServerScope()
 describe('Variable Tests - Cloud', () => {
     forEachSDK((name) => {
         let url: string
-        const capabilities: string[] = SDKCapabilities[name]
+        const capabilities: string[] = SDKCapabilities[name] || []
```

Comment:
this shouldnt need a default it would be an error for it to not find capabilities in here